src/solver/AdvectionSolver.cpp

```diff
@@ -10,7 +10,19 @@
 #include "../Domain.h"
 #include "SolverSelection.h"
 
-AdvectionSolver::AdvectionSolver(FieldController *field_controlller) {
+AdvectionSolver::AdvectionSolver(FieldController *field_controlller) :
+    m_u_lin(FieldType::U,
+                Parameters::getInstance()->get_real("initial_conditions/w_lin"),
+                0,
+                Domain::getInstance()->get_size()),
+    m_v_lin(FieldType::V,
+                Parameters::getInstance()->get_real("initial_conditions/w_lin"),
+                0,
+                Domain::getInstance()->get_size()),
+    m_w_lin(FieldType::W,
+                Parameters::getInstance()->get_real("initial_conditions/w_lin"),
+                0,
+                Domain::getInstance()->get_size()) {
```

Comment:
I think its more readable if w_lin is in an extra line. furthermore, in every line stands w_lin, that is not correct.

Reply:
I agree. done in a821240